pkg/cmd/pipeline_list.go

```diff
@@ -0,0 +1,86 @@
+// Copyright © 2019 The Knative Authors.
```

Comment:
I think we will need to update that at some point :sweat_smile: 
cc @bobcatfish

Reply:
:) 
I modified YOUR NAME <email address> (default) to The Knative Authors so that the same script that changes tektoncd can be run against `cli` as well.